backend/users/crud.py

```diff
@@ -74,17 +70,23 @@ async def get_current_user(token: str = Depends(oauth2_scheme)):
         detail="Could not validate credentials",
         headers={"WWW-Authenticate": "Bearer"},
     )
-    try:
-        payload = jwt.decode(token, os.environ['JWT_SECRET'], algorithms=['HS512'])
-        email: str = payload.get("email")
-        if email is None:
-            raise credentials_exception
-        token_data = schemas.TokenData(email=email)
-    except PyJWTError:
+    token_data = get_token_data(token=token)
```

Comment:
just to check, there is no need to specify the parameter name here right?

Reply:
Yep, I'm just silly that way 😛 (I prefer to use keyword arguments whenever possible)